src/ui/Views/Organisations/Index.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@model OrganisationListViewModel
+
+@{
+    Layout = "_Layout";
```

Comment:
Can we add a page title here, please? `@organisation.OrganisationName`

Reply:
All done